Project.toml

```diff
@@ -6,9 +6,14 @@ version = "2.0.0"
 [deps]
 DBInterface = "a10d1c49-ce27-4219-8d33-6db1a4562965"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
 MySQL = "39abe10b-433b-5dbd-92d4-e302a9df00cd"
+Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
+SafeTestsets = "1bc83da4-3b8d-516f-aca4-4fe02f6d838f"
```

Comment:
The test dependencies must not be in the library's Project (they are not dependencies of SearchLightMySQL itself). The `/test` folder is treated as a separate project and that's where we add the test dependencies. Please see for example how it's done for Genie.jl.

Reply:
In the Postgres adapter you've said that you want have the Project.toml not in the test folder. And the test task in CI needs the Pkg and SafeTestsets to work properly. 
If we work out the tests as a separat project inside the test folder I can change this back. Should I?

Reply:
1/ I'd be surprised to have said that. Can you double check/show me?
2/ CI needs them to be in the package's Project file?
3/ Yes, that is the recommended structure for testing Julia packages.

Reply:
1/ See https://github.com/GenieFramework/SearchLightPostgreSQL.jl/pull/9#discussion_r778891555
As you have commented above I will change the structure of the test folder back to the structure needed for a project and will commit it if it's done.